derive/Cargo.toml

```diff
@@ -19,3 +19,8 @@ quote = "1.0"
 syn = { version = "1.0", features = ["derive", "visit", "visit-mut", "extra-traits"] }
 proc-macro2 = "1.0"
 proc-macro-crate = "1"
+
+[features]
+default = []
+# Include code docs in type metadata.
+docs = []
```

Comment:
Wouldn't we want the docs to be included by default? In the steady-state future scenario we'll want the full metadata I think, and excluding it is something we want to do only while figuring out where to store it yes?